server/src/main/java/server/bazel/cli/Bazel.java

```diff
@@ -0,0 +1,36 @@
+package server.bazel.cli;
+
+import java.util.Optional;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import server.dispatcher.CommandDispatcher;
+import server.dispatcher.CommandOutput;
+
+/**
+ * A wrapper around the Bazel Server Commands. This allows callers to invoke Bazel Server commands
+ */
+public final class Bazel {
+    private static final Logger logger = LogManager.getLogger(Bazel.class);
+    private static final CommandDispatcher dispatcher = CommandDispatcher.create("bazel-command-dispatcher");
+
+    /**
+    * Creates an instance of a Bazel.
+    */
+    private Bazel() {
+    }
+
+    public static void getBuildTargets() {
```

Comment:
Static functions may make this class hard to use in unit tests, makes it so we can't use Mockito (to the best of my knowledge). Maybe we ought to use instance methods so we can do:

```
public class SomeClassThatDependsOnBazel {
  public SomeClassThatDependsOnBazel(Bazel bzl) { ... }

  public void something()
  {
     bzl.getBuildTargets()
  }
}

...

void testSomething() {
    Bazel mockedBazel = Mockito.mock(Bazel.class);
    SomeClassThatDependsOnBazel something = new SomeClassThatDependsOnBazel(mockedBazel);
    ...
}
```

But I'm not a java buf, so someone correct me if there's a different way to do this

Reply:
If we need to mock a static method we can use PowerMock on top of Mockito which allows the mocking of a static method.

Reply:
Perfect